modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Schemas/publishednodesschema.json

```diff
@@ -0,0 +1,184 @@
+                  },
+                  "DisplayName": {
+                    "type": "string"
+                  },
+                  "DataSetFieldId": {
+                    "type": "string"
+                  },
+                  "OpcSamplingInterval": {
+                    "type": "integer"
+                  },
+                  "OpcPublishingInterval": {
+                    "type": "integer"
+                  },
+                  "HeartbeatInterval": {
+                    "type": "integer"
+                  },
+                  "HeartbeatIntervalTimespan": {
+                    "type": "string"
+                  },
+                  "SkipFirst": {
+                    "type": "boolean"
+                  }
+                },
+                "required": [
+                  "Id"
+                ]
+              }
+            ]
+          }
+        },
+        "required": [
+          "EndpointUrl",
+          "OpcNodes"
+        ]
+      },
+      {
+        "type": "object",
+        "properties": {
+          "DataSetWriterId": {
+            "type": "string"
+          },
+          "DataSetWriterGroup": {
+            "type": "string"
+          },
+          "DataSetPublishingInterval": {
+            "type": "string"
+          },
+          "EncryptedAuthPassword": {
+            "type": "string"
+          },
+          "OpcAuthenticationPassword": {
+            "type": "string"
+          },
+          "EndpointUrl": {
+            "type": "string",
+            "format": "uri",
+            "pattern": "opc.tcp?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+"
+          },
+          "UseSecurity": {
+            "type": "boolean"
+          },
+          "OpcNodes": {
+            "type": "array",
+            "items": [
+              {
+                "type": "object",
+                "properties": {
+                  "ExpandedNodeId": {
+                    "type": "string",
+                    "pattern": "(^nsu=http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*(),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+/;(i=(\\d+)$))|(^nsu=http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*(),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+/;(s=([\\x00-\\x7F]|([\\xC2-\\xDF]|\\xE0[\\xA0-\\xBF]|\\xED[\\x80-\\x9F]|(|[\\xE1-\\xEC]|[\\xEE-\\xEF]|\\xF0[\\x90-\\xBF]|\\xF4[\\x80-\\x8F]|[\\xF1-\\xF3][\\x80-\\xBF])[\\x80-\\xBF])[\\x80-\\xBF])+$))|(^nsu=http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*(),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+/;(g=([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})$))|(^nsu=http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*(),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+/;(b=(?:[A-Za-z\\d+/]{4})*(?:[A-Za-z\\d+/]{3}=|[A-Za-z\\d+/]{2}==)$))"
```

Comment:
Please add a comment either in this JSON file (if supported) or in a markdown file near it (let's say `publishednodesschema.md`) with explanation or a few samples of valid values for `ExpandedNodeId` field.

Reply:
Will address pending response on https://github.com/Azure/Industrial-IoT/pull/1305#discussion_r702944645